pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -235,10 +235,26 @@ def test_SingleCells_count():
 
 def test_load_compartment():
     loaded_compartment_df = ap.load_compartment(compartment="cells")
-    pd.testing.assert_frame_equal(loaded_compartment_df, cells_df)
+    pd.testing.assert_frame_equal(
+        loaded_compartment_df, cells_df, check_dtype=False)
 
     # Test non-canonical compartment loading
-    pd.testing.assert_frame_equal(new_compartment_df, ap_new.load_compartment("new"))
+    pd.testing.assert_frame_equal(
+        new_compartment_df, ap_new.load_compartment("new"), check_dtype=False)
+
+
+def test_sc_count_sql_table():  # or something
+    # Iterate over initialized compartments
+    for compartment in ap.compartments:
+        result_row_count = ap.count_sql_table_rows(table=compartment)
+        assert result_row_count == 100
+
+
+def test_get_sql_table_col_names():  # or something
```

Comment:
```suggestion
def test_get_sql_table_col_names():
```

Please also assert `feat_cols`

Reply:
For this I have no consistent ground truth.

Reply:
won't this (or something close) work?

```python
assert feat_cols == [f"{compartment}_a", f"{compartment}_b", f"{compartment}_c", f"{compartment}_d"]
```

Reply:
Cytoplasm breaks the cycle as it has some additional things. But I could do:

```
assert f"{compartment.capitalize()}_a" in feat_cols
```